### DISCUSSION TITLE
Only edit page of resource for specific record

### DISCUSSION THREAD

**Discussion by TheBlackJJoker** [filamentphp/filament#11326] [Help]
### Package

Panel builder

### Package Version

v3

### How can we help you?

Hello, i need to make page with edit form for specyfic record id of model. 

I do not know how to do it in filament. Please, help me

**Comment by leandrocfe** [comment]
If you are using Resources, you could use [Policies](https://filamentphp.com/docs/3.x/panels/resources/getting-started#authorization) like

```php
//app/Policies/ArticlePolicy.php
public function update(User $user, Article $article): bool
{
    return $article->id === 1;
}
```
